resources/user-agents/browsers/opera-mini/opera-mini-8-0-on.json

```diff
@@ -139,24 +139,6 @@
           "platforms": [
             "JAVA"
           ]
-        },
-        {
```

Comment:
These got moved to their own `opera-mini-ios.json` file because I felt there were too many versions in this file (Opera Mini iOS is only at version 14 right now: https://itunes.apple.com/app/id363729560?mt=8&pt=341230&ct=ose_google_via_opera_com_mobile_mini_iphone_top)